clients/admin-ui/cypress/support/commands.ts

```diff
@@ -11,7 +11,15 @@ declare global {
        * Custom command to select DOM element by data-testid attribute
        * @example cy.getByTestId('clear-btn')
        */
-      getByTestId(selector: string): Chainable<JQuery<HTMLElement>>;
+      getByTestId(
+        selector: string,
+        options?: Partial<
```

Comment:
Were these useful while debugging? I don't see options being passed in any of the tests.

Reply:
I think at some point I was giving a `timeout` option to try to reduce flakiness. in any case, it's useful to have around for the future, and I probably should written it this way in the first place to be as similar as possible to regular `cy.get` 😅